### PR TITLE
Use parameters correctly

### PR DESCRIPTION
**Pull request by MaximilianMeister**
## Why is this PR needed?

e2e pipeline is broken

## What does this PR do?

It fixes the e2e pipeline

## Anything else a reviewer needs to know?

it's a follow-up for https://github.com/SUSE/skuba/pull/603

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
> LGTM, but not an expert on Jenkins pipeline syntax.

thanks, i checked it against `https://ci.suse.de/pipeline-model-converter/validate`

File: ci/jenkins/pipelines/skuba-e2e-nightly.Jenkinsfile [hunk 1]
```diff
@@ -5,11 +5,9 @@
 pipeline {
    agent { node { label 'caasp-team-private' } }
 
-   properties([
-       parameters([
-           string(name: 'E2E_MAKE_TARGET_NAME', defaultValue: 'all', description: 'The make target to run (only e2e related)')
-       ])
-   ])
+   parameters {
+        string(name: 'E2E_MAKE_TARGET_NAME', defaultValue: 'all', description: 'The make target to run (only e2e related)')
```
Comment (Klaven) [inline ci/jenkins/pipelines/skuba-e2e-nightly.Jenkinsfile]: below you use `e2e` hard coded. i'm not expert in this, just noticed this.

Comment (MaximilianMeister) [reply ci/jenkins/pipelines/skuba-e2e-nightly.Jenkinsfile]: yes, the groovy linter says:

```
WorkflowScript: 28: Expected string literal @ line 28, column 15.
           stage("Run Skuba ${E2E_MAKE_TARGET_NAME} Test") {
                 ^
```

which basically means I can't use a variable in the stage description, but it's not a big deal so i hardcoded just e2e. we have the 'E2E_MAKE_TARGET_NAME' in the label, so it shows up in the UI

**Comment by jordimassaguerpla** [comment]
CI code. Ok to release.

**Pull request merged**